bot/database/models.py

```diff
@@ -19,13 +16,6 @@ class UserProfile:
     note: Optional[str]
 
 
-@dataclass
```

Comment:
jestes pewien ze poodwracales wszystko? strasznie duzo pokasowales XD

Reply:
Nie no chyba jest git przy tym robieniu cooldownu skumałem że tam jest nadwyżka tych structów z resztą jakbym ważny struct wywalił to by nie nie ruszyło nie czarujmy się xD